components/molecule/notification/src/index.js

```diff
@@ -38,6 +38,18 @@ class MoleculeNotification extends Component {
     this.state.show !== nextProps.show && this.toggleShow()
   }
 
+  componentDidMount() {
+    const {show, autoClose: autoCloseTiming} = this.props
```

Comment:
if encapsulated the checked in a function so its not duplicated. Anyway... the name for sure could be more accurated. I accept proposals.

Since autoClose is a function in the component,  I would do the check inside the function following the `tell, dont ask principle`